crates/miden-tx/Cargo.toml

```diff
@@ -27,6 +27,8 @@ miden-objects = { workspace = true }
 miden-processor = { workspace = true }
 miden-prover    = { workspace = true }
 miden-verifier  = { workspace = true }
+# TODO do we want this unconditionally? Is that an assumption we can make?
+miden-stdlib    = { workspace = true }
```

Comment:
If I remove these lines, everything still seems to work - so do we need it? (It could be I'm missing something with this - I do see the usage of stdlib in miden-tx, so not fully sure how this still works. Probably a transitive dependency somehow).